### PR TITLE
wc: mb: Add project function get_hsc_pwr_reading

### PR DESCRIPTION
**Pull request by MouchenHung-QUANTA**
Summary:
- Support HSC sensor read for oem nm sensor read command NetFn: 0x36 Cmd:0xE2.

Test Plan:
- Build Code: PASS

### CODE REVIEW

File: meta-facebook/wc-mb/src/platform/plat_sensor_table.c [hunk 1]
```diff
@@ -254,6 +254,11 @@ bool pal_is_time_to_poll(uint8_t sensor_num, int poll_time)
 	return true;
 }
 
+uint8_t get_hsc_pwr_reading(int *reading)
+{
+	return get_sensor_reading(SENSOR_NUM_PWR_HSCIN, reading, GET_FROM_CACHE);
+}
+
```
Comment (GoldenBug) [inline meta-facebook/wc-mb/src/platform/plat_sensor_table.c:261]: Check to make sure reading isn't NULL using macro recently added.

**Review by GoldenBug — CHANGES_REQUESTED** [review]

File: meta-facebook/wc-mb/src/platform/plat_sensor_table.c (hunk 1 above)
Comment (MouchenHung-QUANTA) [reply meta-facebook/wc-mb/src/platform/plat_sensor_table.c:261]: Normally, There should have a pointer check for input parameters, but in this case, function "get_hsc_pwr_reading()" is just a transporter of "get_sensor_reading()". It already has a certain return value for invalid pointer case. Should I add another return value or just using SENSOR_UNSPECIFIED_ERROR which is the same as "get_sensor_reading()" invalid pointer case, thanks.

Comment (GoldenBug) [reply meta-facebook/wc-mb/src/platform/plat_sensor_table.c:261]: I would add another check here. That way if the function were to change in the future the check won't be forgotten.

Comment (MouchenHung-QUANTA) [reply meta-facebook/wc-mb/src/platform/plat_sensor_table.c]: Understood.
New commit has been updated, please have a look thanks.

**Pull request closed without merging**